tests/test_stream.py

```diff
@@ -0,0 +1,28 @@
+from morphocut import Pipeline
+from morphocut.stream import Slice
+
+import pytest
+
+
+def test_Slice():
+    # Assert that the stream is sliced
+    items = "ABCDEFG"
+
+    with Pipeline() as pipeline:
+        result = Slice(2)()
+
+    stream = list(pipeline.transform_stream(items))
+    obj = stream
+
+    assert obj[0] == 'A'
+    assert obj[1] == 'B'
```

Comment:
I would not call this `stream` anymore, because you converted it to a list. Rather:
```
result = list(...)
assert result == ["A", "B"]
```